File: apps/android/app/src/main/kotlin/com/oasisbio/app/domain/model/Session.kt
package com.oasisbio.app.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class Session(
    val accessToken: String,
    val refreshToken: String,
    val expiresIn: Long,
    val expiresAt: Long,
    val tokenType: String,
    val userId: String
)

File: apps/ios/OasisBio/Shared/Models/Session.swift
import Foundation

struct Session: Codable {
    let accessToken: String
    let refreshToken: String
    let expiresAt: Date
    let userId: String
    
    enum CodingKeys: String, CodingKey {
        case accessToken = "access_token"
        case refreshToken = "refresh_token"
        case expiresAt = "expires_at"
        case userId = "user_id"
    }
}
